src/App.test.js

```diff
@@ -1,14 +1,14 @@
 import React from 'react';
 import { render } from '@testing-library/react';
 import { Provider } from 'react-redux';
-import { store } from './app/store';
+import store from './app/store';
 import App from './App';
 
 test('renders learn react link', () => {
   const { getByText } = render(
     <Provider store={store}>
       <App />
-    </Provider>
+    </Provider>,
   );
 
   expect(getByText(/learn/i)).toBeInTheDocument();
```

Comment:
- [ ] You have only one test and it is failing. Kindly fix it.

  ![akumu-test](https://user-images.githubusercontent.com/36542986/163194411-67e3bb7e-d536-4d03-89b7-891b94681300.jpg)

- [ ] According to the requirements you are expected to write unit tests and integration tests using React Testing Library. Kindly write more tests to meet the requirement.

  ![akumu-test-req](https://user-images.githubusercontent.com/36542986/163194792-cc242880-75db-480a-8ed0-bcf750d25c37.jpg)